subcommands/waves/init.go

```diff
@@ -157,10 +158,11 @@ func doInitWave(cmd *cobra.Command, args []string) {
 WARNING: You created a wave for a target version without static deltas.
 
 We recommend that you generate static deltas for your production targets to optimize
-OTA update download. Consider generating a static delta for targets using: 
+OTA update download. 
+Consider generating a static delta for targets using: 
 $ fioctl targets static-deltas
 
-You can then cancel the wave and create a new one by using a static delta.
+You can then cancel this wave and create a new one for a target with a static delta.
 `)
 
 	}
```

Comment:
This is the most convenient way to comment on the commit message itself.
There is a typo: weave -> wave.